### PR TITLE
More 0.5.0 changes

### PR DESCRIPTION
**Pull request by alexander-yakushev**
The information is in the commits.

### CODE REVIEW

File: deps.edn [hunk 1]
```diff
@@ -2,17 +2,17 @@
  :deps {org.clojure/clojure {:mvn/version "1.11.1" :mvn/scope "provided"}}
 
  :aliases
- {:build {:deps {io.github.clojure/tools.build {:git/tag "v0.9.6"
+ {:build {:deps {io.github.clojure/tools.build {:git/tag "v0.10.0"
                                                 :git/sha "8e78bcc"}
```
Comment (Menschenkindlein) [inline deps.edn]: Don't forget to change the sha: `3a2c484`

https://github.com/clojure/tools.build/releases/tag/v0.10.0

File: src/omniconf/core.clj [hunk 2]
```diff
@@ -30,17 +30,15 @@
...
-
 (defn set-logging-fn
   "Change `println` to a custom logging function that Omniconf will use."
   [fn]
   (reset! logging-fn fn))
 
-(defn enable-functions-as-defaults
-  "Allow invoking functions passed to :default field for the options."
+(defn ^:deprecated enable-functions-as-defaults
+  "WARNINGThis function Allow invoking functions passed to :default field for the options."
```
Comment (Menschenkindlein) [inline src/omniconf/core.clj]: Unclear changes in the docstring.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,457 @@
+# Omniconf [![CircleCI](https://img.shields.io/circleci/build/github/grammarly/omniconf/master.svg)](https://dl.circleci.com/status-badge/redirect/gh/grammarly/omniconf/tree/master) [![](https://img.shields.io/clojars/dt/com.grammarly/omniconf?color=teal)](https://clojars.org/com.grammarly/omniconf) [![](https://img.shields.io/badge/-changelog-blue.svg)](CHANGELOG.md)
```
Comment (Menschenkindlein) [inline README.md:1]: So the plan is to have both `md` and `org`, right?

File: deps.edn (hunk 1 above)
Comment (alexander-yakushev) [reply deps.edn]: Good catch, thank you!

File: README.md (hunk 3 above)
Comment (alexander-yakushev) [reply README.md:1]: My bad, forgot to remove the org file.

**Comment by alexander-yakushev** [comment]
Comments addressed.

**Review by Menschenkindlein — APPROVED** [review]
Looks good 👍

**Pull request merged**